### PR TITLE
Add model description

### LINKED ISSUE #40: Write description of the Stan model
There should be a verbal description of the statistical model for inferring P2RA conversion factors in the README (or some other docs).

### PR DESCRIPTION
**Pull request by dp-rice**
Resolves #40. Adds a Markdown file describing the current model and its limitations.

### CODE REVIEW

**Comment by dp-rice** [comment]
@simonleandergrimm, this is for our meeting tomorrow. Would you mind giving it a quick skim today and letting me know if there's anything so unclear/confusing that it would be worth fixing before the morning?

**Comment by dp-rice** [comment]
BTW, this is the best place to see the rendered markdown:
https://github.com/naobservatory/p2ra/blob/dan-model-description/model.md

File: model.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+- The expected log-prevalence of the viral clade in the population contributing to each sample, $\mu_j$.
+- An estimate of the uncertainty in the log-prevalence, represented as a standard deviation $\sigma$, assumed to be the same across samples.
+
+### Parameters
+
+When we run the program, Stan uses a [Hamiltonian Monte Carlo](https://en.wikipedia.org/wiki/Hamiltonian_Monte_Carlo) algorithm to sample from the posterior distribution of the model parameters. 
+They are:
+
+- The true log-prevalence in the population contributing to each sample, $\theta_j$.
+- The coefficient linking the log-prevalence to the expected number of viral reads, $b$. Note that this is a scalar quantity, assumed to be the same across all samples.
```
Comment (simonleandergrimm) [inline model.md:53]: Across all samples for one viral clade right?

File: model.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+We assume that the read counts follow a negative binomial distribution and are independent (conditional on the parameters):
+
+$$
+y_i \sim NB(n_i \exp(b + \theta_i), \phi),
+$$
+
+using the (mean, reciprocal overdispersion) parameterization.
+Note that expected number of reads is proportional to the total number of reads times the exponentiated log-prevalence.
+The constant of proportionality is $e^b$.
+In other words, $b$ is the intercept term of a negative binomial regression with a log link function.
```
Comment (simonleandergrimm) [inline model.md:67]: Flagging that I don't fully grasp what this sentence means.

File: model.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+We give our true log-prevalence parameters independent Gaussian priors centered on our log-prevalence estimates:
+
+$$
+\theta_i \sim Normal(\mu_i, \sigma).
+$$
+
+This is equivalent to assuming a flat prior on the log-abundance and updating with Gaussian-likelihood to our prevalence data.
+
+We put a weakly informative Gaussian prior on $b$.
+(We can center this at zero by normalizing abundance inputs.)
```
Comment (simonleandergrimm) [inline model.md:79]: Can you expand on why you can do this through normalization?

File: model.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+### Limitations and future directions
+
+Our biggest current limitation is that we model the true prevalences as independent of one another.
+It would be more realistic for them to have shared as well as independent components of variation.
+For example, our prevalence estimates may be systematically biased in one direction or another and that would induce a correlated effect on all of the samples.
+The amount of shared variation between samples might also depend on features they have in common such as a shared sampling location or similar sampling times.
+(Issue [#59](https://github.com/naobservatory/p2ra/issues/59) outlines a simple way forward here.)
+
+A related issue is that we may have prevalence data that is more relevant for some samples than for others, but is not perfectly aligned with the sampling times and locations.
```
Comment (simonleandergrimm) [inline model.md:92]: Why is there a but clause here?

File: model.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+In this case, we may want to have a stochastic process model (e.g., a Gaussian process) of the true prevalence that can be updated with our prevalence estimates and then emit the parameters for the negative binomial regressions.
```
Comment (simonleandergrimm) [inline model.md:93]: Explaining this again in different words could be good.

File: model.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+As mentioned in the [Model](#model) section, it would also be useful to extend the negative binomial regression to include predictors other than sample identity.
+By including sample processing methods, or viral characteristics as predictors, we could combine information across studies and viruses to develop a general model of MGS abundance. 
+
+Finally, we currently do not do any evaluation of model fit or sensitivity to prior assumptions.
+We will develop a workflow to check that the model is appropriate for the data and to improve it if not.
```
Comment (simonleandergrimm) [inline model.md:99]: What does such a workflow broadly look like>

**Review by simonleandergrimm — APPROVED** [review]
Overall looks good, thanks for writing this up! I would give a bit more context on some of the model choices, though I also understand that we discussed some of the choices already.

**Comment by dp-rice** [comment]
Thanks, @simonleandergrimm!

**Pull request merged**